.gitignore

```diff
@@ -27,3 +27,4 @@ chrome-user-data
 .idea
 *.iml
 .vscode
+yarn.lock
```

Comment:
Why would you ignore yarn.lock? The whole point of it is to be included to ensure specific versions of things are installed.